src/app/services/views/ServiceDetailTabsView.vue

```diff
@@ -89,6 +89,13 @@ function getNavTabs(serviceInsight: ServiceInsight): NavTab[] {
 
   return routes
     .filter((route) => {
+      if (
+        (serviceInsight.serviceType === undefined || serviceInsight.serviceType === 'internal') &&
```

Comment:
If I understand this `internal` thing correctly. It would be good to just set `serviceType` to equal `internal` as a default when it comes in the the HTTP API, then we would know that it is always a `string` not `string | undefined` and not have to worry about these things in the rest of the application.

Reply:
This is a case were really this shouldn’t even be possible to be omitted on the API in the first place, but as far as I’m aware it can’t be changed there because of our use of protobuf.

Not a big point in changing this now if we’re eventually streamlining this sort of data on the DataSource level.

Reply:
Yeah gotcha, no action here rn. but in relation to the ongoing data layer discussions, if we had our own data layer to smooth these things out/hide this sharp edge from our application we could until it was resolve in the backend